src/containers/App/App.js

```diff
@@ -1,10 +1,17 @@
 import React, { Component } from 'react';
+import { SearchBar } from '../SearchBar/SearchBar';
+import { Filters } from '../Filters/Filters';
+import { EventContainer } from '../EventContainer/EventContainer';
+import { NavBar } from '../../components/NavBar/NavBar';
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <h1>Ventures</h1>
+      <div className="App"> 
+        <SearchBar />
+        <Filters />
+        <EventContainer />
+        <NavBar />
```

Comment:
For my BikeSee app, it was pretty easy to start with the NavBar on top for the desktop version and then just update the positioning for mobile. Might be easier to do this by rendering the NavBar container first.

Reply:
That's a great call! Will do.